pom.xml

```diff
@@ -263,7 +263,7 @@
                                 <!-- Oracle image - we aim to use same version as Dev Services for Oracle so that we only download one image -->
                                 <oracle.image>docker.io/gvenzl/oracle-free:23-slim-faststart</oracle.image>
                                 <db2.image>quay.io/quarkusqeteam/db2:11.5.7.0</db2.image>
-                                <mongodb.image>docker.io/bitnami/mongodb:5.0</mongodb.image>
+                                <mongodb.image>docker.io/library/mongo:5.0</mongodb.image>
```

Comment:
Did you try this on podman? It also uses randomly assigned UID, so it may have the same problem on baremetal tests

Reply:
I created my own instance on OpenStack with Podman and it worked. I presume it goes down to how filesystem is shared or something, but I definitely don't know that (nor I care TBH).